src/reducers/createArticleReducer.js

```diff
@@ -0,0 +1,37 @@
+import { createArticleType } from '../actions/createArticles';
+import { resetArticleMessage } from '../actions/clearMessage';
+
+const initialState = {
+  loading: false,
+  message: false,
+};
+
+/**
+ * create an article reducer
+ *
+ * @export
+ * @param {object} state - initial state
+ * @param {object} action - to update state
+ * @returns {object}
+ */
+
+export default function createArticleReducer(state = initialState, action = {}) {
+  const {
+    type, message
+  } = action;
+  switch (type) {
+    case createArticleType.loading:
+      return { ...state, loading: true, message: false };
+    case createArticleType.success:
```

Comment:
If this is here because nothing happens in the store when an article is created, then maybe we should remove it. What do you think?

Reply:
resolved